releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -6,10 +6,10 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 * Updated the readme 
 
 ### v1.1.3 - December, 10 2020
-* Erorr handling in the case where, required parameters are missing 
+* Improved error message when required parameters are not provided 
 
 ### v1.0.4 - November, 11 2020

```

Comment:
I thought about this a bit more. Patch versions aren't visible to end users. Since there were no changes in this release I think we would be fine just omitting it.

Reply:
How come this comment resolved with no reply or action taken?